charts/testkube-enterprise/Chart.yaml

```diff
@@ -16,6 +16,11 @@ dependencies:
   - name: testkube-worker-service
     version: 1.29.2
     repository: file://../testkube-worker-service
+  - name: testkube
+    alias: testkube-agent
+    version: ~1.17.24
```

Comment:
What's your opinion on `~` for our own sub-charts? I'd refrain from using it on external ones.

Reply:
We are checking in `Chart.lock` file, not sure this would be compatible with that approach.